stm32-modules/include/common/core/thermistor_conversion.hpp

```diff
@@ -45,6 +45,7 @@ struct Conversion {
     /** First is After, second is Before (after - 1) */
     using TableEntryPair = std::pair<TableEntry, TableEntry>;
     using TableResult = std::variant<TableEntryPair, TableError>;
+    static constexpr double HEATER_PAD_DISCONNECT_SAFETY_LIMIT_ADC = 3642.3; //0C equivalent
```

Comment:
This should be a configurable parameter of the thermistor conversion class instead of being hard coded. The same driver is used on the thermocycler with a different ADC depth and a different resistor ladder.